### ISSUE TITLE
solution doesn't cover first step

### ISSUE THREAD

**Issue by BuffaloRob** [learn-co-curriculum/js-ajax-callbacks-lab#27]
The solution has nothing in it about the link that is supposed to be made in step one. The wording on this step is also confusing, it says to make a link but the link needs to accept an input, shouldn't this be a form (with an input) instead of a link?

**Comment by efl7a** [comment]
I agree with BuffaloRob.  I was looking forward to see how you handled this in the solution as I would have and think that it should be a form with a search input.
